classes/static/toolbar.css

```diff
@@ -0,0 +1,120 @@
+/* Loading Modal */
+.autoptimize-loading
+{
+	display: none;
+	position: fixed;
+	background-color: rgba(102, 102, 102, 0.8);
+	background-image: url('loading.gif');
+	background-position: center;
+	background-repeat: no-repeat;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	z-index: 9000000000;
+}
+
+/* Toolbar Font Colors */
+#wp-admin-bar-autoptimize .white
+{
+	color: #EEE;
+}
+
+#wp-admin-bar-autoptimize .green
+{
+	color: #02CA02;
+}
+
+#wp-admin-bar-autoptimize .orange
+{
+	color: #EC9103;
+}
+
+#wp-admin-bar-autoptimize .red
+{
+	color: #EA1919;
+}
+
+/* Toolbar Bullet Icons */
+#wp-admin-bar-autoptimize.bullet-green .ab-icon::before,
+#wp-admin-bar-autoptimize.bullet-green:hover .ab-icon::before
+{
+	content: "\f159";
+	color: #02CA02;
+	font-size: 14px;
+}
+
+#wp-admin-bar-autoptimize.bullet-orange .ab-icon::before,
+#wp-admin-bar-autoptimize.bullet-orange:hover .ab-icon::before
+{
+	content: "\f159";
+	color: #EC9103;
+	font-size: 14px;
+}
+
+#wp-admin-bar-autoptimize.bullet-red .ab-icon::before,
+#wp-admin-bar-autoptimize.bullet-red:hover .ab-icon::before
+{
+	content: "\f159";
+	color: #EA1919;
+	font-size: 14px;
+	-webkit-animation: blink 1s step-end infinite;
+	animation: blink 1s step-end infinite;
+}
+
+@-webkit-keyframes blink { 50% { visibility: hidden; }}
+        @keyframes blink { 50% { visibility: hidden; }}
+
+		
+/* Some cosmetic Toolbar things */
+#wp-admin-bar-autoptimize-default
+{
+	padding-top: 0 !important;
+}
+
+#wp-admin-bar-autoptimize-delete-cache .ab-item
+{
+	cursor: pointer !important;
+	background: #464b50;
+}
+
+#wp-admin-bar-autoptimize-delete-cache .ab-item:hover
+{
+	color: rgba(240,245,250,0.85) !important;
+	background: #B57373 !important;
+}
+
+#wp-admin-bar-autoptimize-cache-info
+{
+	padding-top: 8px !important;
+	padding-bottom: 8px !important;
+}
+
+#wp-admin-bar-autoptimize-cache-info,
+#wp-admin-bar-autoptimize-cache-info .ab-item
+{
+	height: auto !important;
+	cursor: default !important;
+}
+
+#wp-admin-bar-autoptimize-cache-info td + td
+{
+	padding-left: 3px;
+}
+
+#wp-admin-bar-autoptimize-cache-info .ab-item,
+#wp-admin-bar-autoptimize-cache-info .ab-item:hover
+{
+	color: #b4b9be !important;
+}
+
+#wp-admin-bar-autoptimize-cache-info .ab-item > p
+{
+	display: block;
+}
+
+#wp-admin-bar-autoptimize-cache-info .ab-item * 
+{
+	font-size: 11px !important;
+	line-height: 16px !important;
+}
```

Comment:
had a conflict with a theme when in frontend, with the table in the dropdown visible, this fixed that;```

#wp-admin-bar-autoptimize table th td 
{
	border: 0px;
}
```